### PR TITLE
Refactor EmployeeGUI and CustomerGUI

### PR DESCRIPTION
**Pull request by chillyssa**
This PR Introduces several refactoring updates: 
- removed CustomerGUI file as we don't use it anymore in our application 
- formatted EmployeeGUI to align with formatting guide 
- refactored variable declarations to declare then initialize inside the constructor of the class 
- removed unnecessary .setText methods to reduce code 
- updated file with more informative comments 
- removed initialization of window object to just instantiate the needed Inventory/Order page

### CODE REVIEW

**Review by snowjtSD — APPROVED** [review]
also looks good, thanks again.

**Comment by chillyssa** [comment]
I missed a comment in EmployeeGUI.java. I have re-pushed a change and re-requested review from @snowjtSD

File: app/src/main/java/com/yagni/view/EmployeeGUI.java [hunk 1]
```diff
@@ -8,154 +8,186 @@
 import com.yagni.controller.*;
```
Comment (S4kRiPp4) [inline app/src/main/java/com/yagni/view/EmployeeGUI.java:7]: delete whitespace on line 7. move import com.yagni up to line 7

File: app/src/main/java/com/yagni/view/EmployeeGUI.java [hunk 2]
```diff
@@ -8,154 +8,186 @@
...
+			// Passing the db connection with new object to render Employee GUI page
+			new ManageInvGUI(linkDB);
+		}
+		if (e.getSource() == orderButton) {
+			// On click of order button dispose this frame
+			frame.dispose();
+
+			// Passing the db connection with new object to render Orders GUI page
+			new OrdersGUI(linkDB);
+
```
Comment (S4kRiPp4) [inline app/src/main/java/com/yagni/view/EmployeeGUI.java]: delete whitespace at 182, moving curly from 184 up a line

File: app/src/main/java/com/yagni/view/EmployeeGUI.java [hunk 3]
```diff
@@ -8,154 +8,186 @@
...
+		frame.setResizable(false);
+
+		// add the header, body and footer panels to the frame with BorderLayout manager
+		frame.add(headerPanel, BorderLayout.NORTH);
+		frame.add(bodyPanel, BorderLayout.CENTER);
+		frame.add(footerPanel, BorderLayout.SOUTH);
+
+		// display all content to the GUI
+		frame.setVisible(true);
+
```
Comment (S4kRiPp4) [inline app/src/main/java/com/yagni/view/EmployeeGUI.java]: delete whitespace at 163, moving line 164 curly up

File: app/src/main/java/com/yagni/view/ManageInvGUI.java [hunk 4]
```diff
@@ -5,369 +5,474 @@
...
-				deleteObj.delete(linkDB);  //calls the delete method and passes in the database connection.
-				refreshProducts();  
-				clearTextFields();
+                // Create object to call delete, passing through deleteID and db connection
+                Delete deleteObj = new Delete(deleteID);
+                deleteObj.delete(linkDB);
+
+                // Refresh products Table
+                refreshProducts();
+
```
Comment (S4kRiPp4) [inline app/src/main/java/com/yagni/view/ManageInvGUI.java]: delete whitespace here, moving the 3 following curly braces up a line

File: app/src/main/java/com/yagni/view/ManageInvGUI.java [hunk 5]
```diff
@@ -5,369 +5,474 @@
...
+        inventoryFrame.setResizable(false);
+
+        // Add the header, body and footer panels to the frame
+        inventoryFrame.add(headerPanel, BorderLayout.NORTH);
+        inventoryFrame.add(bodyPanel);
+        inventoryFrame.add(footerPanel, BorderLayout.SOUTH);
+
+        // Displays all content to the GUI
+        inventoryFrame.setVisible(true);
+
```
Comment (S4kRiPp4) [inline app/src/main/java/com/yagni/view/ManageInvGUI.java]: delete whitespace at 270, moving closing curly up a line

File: app/src/main/java/com/yagni/view/EmployeeGUI.java (hunk 1 above)
Comment (chillyssa) [reply app/src/main/java/com/yagni/view/EmployeeGUI.java:7]: removed!

File: app/src/main/java/com/yagni/view/EmployeeGUI.java (hunk 2 above)
Comment (chillyssa) [reply app/src/main/java/com/yagni/view/EmployeeGUI.java]: Removed!

File: app/src/main/java/com/yagni/view/EmployeeGUI.java (hunk 3 above)
Comment (chillyssa) [reply app/src/main/java/com/yagni/view/EmployeeGUI.java]: removed!

File: app/src/main/java/com/yagni/view/ManageInvGUI.java (hunk 4 above)
Comment (chillyssa) [reply app/src/main/java/com/yagni/view/ManageInvGUI.java]: Removed whitespace on 374

File: app/src/main/java/com/yagni/view/ManageInvGUI.java (hunk 5 above)
Comment (chillyssa) [reply app/src/main/java/com/yagni/view/ManageInvGUI.java]: Removed Whitespace line 270

**Comment by chillyssa** [comment]
@S4kRiPp4 resolved your comments and pushed changes. Please re-review!

**Review by S4kRiPp4 — APPROVED** [review]
Thanks for changes, everything looks nice and organized. Nice comments too

**Pull request merged**